specification/Metadata/Semantics/README.md

```diff
@@ -48,6 +48,8 @@ Units for all linear distances are meters, and all angles are radians.
 Semantic|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Description
 --|--|--
 `TILESET_FEATURE_ID_SETS`|<ul><li>Type: `STRING`</li><li>Array: `true`</li></ul>|The union of all the feature ID sets in glTF content using the [`EXT_mesh_features`](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features) and [`EXT_instance_features`](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_instance_features) extensions.
+`TILESET_CRS_GEOCENTRIC`|<ul><li>Type: `STRING`</li></ul>|The geocentric coordinate reference system (CRS) of the tileset. Known values include:<ul><li>`"EPSG:4978"` - WGS 84</li><li>`"EPSG:7656"` - WGS 84 (G730)</li><li>`"EPSG:7658"` - WGS 84 (G873)</li><li>`"EPSG:7660"` - WGS 84 (G1150)</li><li>`"EPSG:7662"` - WGS 84 (G1674)</li><li>`"EPSG:7664"` - WGS 84 (G1762)</li><li>`"EPSG:9753"` - WGS 84 (G2139)</li></ul>`region` bounding volumes are assumed to use the same reference ellipsoid as the geocentric coordinate reference system specified here.<br><br>For more details on coordinate reference systems in 3D Tiles, see [Coordinate Reference System (CRS)](../../#coordinate-reference-system-crs).
+`TILESET_CRS_COORDINATE_EPOCH`|<ul><li>Type: `SCALAR`</li><li>Component Type: `FLOAT32` or `FLOAT64`</li></ul>|The coordinate epoch for coordinates that are referenced to a dynamic CRS such as WGS 84. Coordinates include glTF vertex positions after transforms have been applied — see [glTF transforms](https://github.com/CesiumGS/3d-tiles/tree/main/specification#gltf-transforms). Expressed as a decimal year (e.g. `2019.81`). See [WKT representation of coordinate epoch and coordinate metadata](http://docs.opengeospatial.org/is/18-010r7/18-010r7.html#128) for more details.
```

Comment:
Why is this a float and not a string or a VEC2 of UINTN? The WKT spec describes it as `<unsigned integer> . <unsigned integer>`. A float value would result in a loss of precision.

The example of `2019.81` is actually a case where this is not exactly representable:

![image](https://user-images.githubusercontent.com/8422414/156400890-22ac281d-4f6c-4909-8989-3c50c6552e4d.png)

Reply:
Good point. `STRING` seems like a better choice here.